src/linear_algebra/determinant.lean

```diff
@@ -368,6 +368,25 @@ end
 lemma basis.is_unit_det (e' : basis ι R M) : is_unit (e.det e') :=
 (is_basis_iff_det e).mp ⟨e'.linear_independent, e'.span_eq⟩
 
+/-- Any alternating map to `R` where `ι` has the cardinality of a basis equals the determinant
+map with respect to that basis, multiplied by the value of that alternating map on that basis. -/
+lemma alternating_map.eq_smul_det (f : alternating_map R M R ι) : f = f e • e.det :=
+begin
+  refine basis.ext_alternating e _,
+  intro i,
+  by_cases h : function.injective i,
+  { let σ : equiv.perm ι := equiv.of_bijective i (fintype.injective_iff_bijective.1 h),
+    change f (e ∘ σ) = (f e • e.det) (e ∘ σ),
+    simp [alternating_map.map_perm, basis.det_self] },
+  { rw function.injective at h,
```

Comment:
This part could go into `basis.ext_alternating` too, right? You only need to show equality on the injective `v`s a you know the others are zero.

Reply:
I've moved injectivity into the hypotheses for `basis.ext_alternating` (and, thus, the proof for the non-injective case into the proof of that lemma) as suggested.